### DISCUSSION TITLE
Possible docs typo on "Using Turborepo with GitHub Actions"

### DISCUSSION THREAD

**Discussion by pqt** [vercel/turborepo#1307] [Help] [answered]
On the following page for GitHub Actions (See: https://turborepo.org/docs/ci/github-actions).

The following turbo.json is provided as an example.
```json
{
  "$schema": "https://turborepo.org/schema.json",
  "baseBranch": "origin/main",
  "pipeline": {
    "build": {
      "dependsOn": ["^build"],
      "outputs": []
    },
    "test": {
      "dependsOn": ["^build"],
      "outputs": []
    },
  }
}
```

`pipeline.build` depends on `^build`. Is that ... required? Or is it a typo?

I don't see it addressed anywhere so everything leads me to believe it's a typo, but I am also a complete novice to turborepo so I wanted to ask before I made a PR.

**Comment by ThibautMarechal** [comment, marked as answer]
Everything is fine. The '^' mean that to build a package, its dependencies need to be built before.
(https://turborepo.org/docs/core-concepts/pipelines#topological-dependency)
